ted_sws/core/adapters/cmd_runner.py

```diff
@@ -8,8 +8,7 @@
     EventWriterToMongoDBHandler
 from ted_sws.event_manager.adapters.event_handler_config import CLILoggerConfig
 from ted_sws.event_manager.adapters.event_logger import EventLogger
-from ted_sws.event_manager.adapters.log import SeverityLevelType
-from ted_sws.event_manager.adapters.logger import LOG_ERROR_TEXT, LOG_SUCCESS_TEXT
+from ted_sws.event_manager.adapters.log import SeverityLevelType, LOG_ERROR_TEXT, LOG_SUCCESS_TEXT
```

Comment:
This module seems to be something that belongs to the entry points, not adapters